management/Dockerfile.entra-test

```
@@ -0,0 +1,67 @@
+
+# Prime the module cache separately so source edits don't invalidate it.
+COPY go.mod go.sum ./
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    go mod download
+
+# Copy the entire repo into the build stage. This is a dev-only harness
+# image; no secrets are expected to live in the source tree and there is no
+# production layer that ships this COPY — see management/Dockerfile for the
+# release build path used by goreleaser.
+COPY . . # NOSONAR
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)